<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommitPropertyDialog</class>
 <widget class="QDialog" name="CommitPropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commit Property</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ReadOnlyLineEdit" name="lineEdit_description">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ReadOnlyLineEdit" name="lineEdit_date">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ReadOnlyLineEdit" name="lineEdit_author">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Mail</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ReadOnlyLineEdit" name="lineEdit_mail">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Commit ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ReadOnlyLineEdit" name="lineEdit_commit_id">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Parent IDs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ReadOnlyPlainTextEdit" name="plainTextEdit_parent_ids">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_checkout">
       <property name="text">
        <string>Checkout</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReadOnlyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ReadOnlyLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ReadOnlyPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>ReadOnlyPlainTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_description</tabstop>
  <tabstop>lineEdit_date</tabstop>
  <tabstop>lineEdit_author</tabstop>
  <tabstop>lineEdit_mail</tabstop>
  <tabstop>lineEdit_commit_id</tabstop>
  <tabstop>plainTextEdit_parent_ids</tabstop>
  <tabstop>pushButton_checkout</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CommitPropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
